jekyll/_cci2/hello-world-macos.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: classic-docs
+title: "Hello World On MacOS"
+short-title: "Hello World On MacOS"
+description: "First macOS project on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
+This document describes how to get started with continuous integration on **macOS
+build environments** on CircleCI. If you still need to get acquainted with
+CircleCI, it is recommended to checkout the [getting started guide]({{ site.baseurl }}/2.0/getting-started).
+
+## Prerequisites
+
+To follow along with this document you will need:
+
+- An [account](https://circleci.com/signup/) on CircleCI.
+- A subscription to a [paid plan](https://circleci.com/pricing/#build-os-x) to enable building on the macOS executor.
+- An Apple computer with XCode installed on it (if you want to open the example project).
+
+## Overview Of The macOS Executor
+
+The macOS build environment (or `executor`) is used for iOS and macOS
+development, allowing you to test, build, and deploy macOS and iOS applications on
+CircleCI. The macOS executor runs jobs in a macOS environment on a VM and provides access to iOS simulators.
```

Comment:
```suggestion
CircleCI. The macOS executor runs jobs in a macOS environment and provides access to iPhone, iPad, Apple Watch and Apple TV simulators.
```

I'd remove the reference to the VM here – I think that introduces complexity that's not needed at this point.